app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+
+
+def move_file(command):
+    if command.split()[0] == "mv":
+        file = command.split()[1]
+        path = command.split()[2]
```

Comment:
you use `split` 3 times while you can use it once

Reply:
changed